gulpfile.js

```diff
@@ -48,49 +48,39 @@ gulp.task(
 gulp.task(
   'build:browser',
   gulp.series('lint:src', function buildBrowser() {
-    return (
-      gulp
-        .src('src/browser.js')
-        .pipe(plumber())
-        .pipe(
-          plugins.webpack({
-            output: { library: 'StellarSdk' },
-            module: {
-              loaders: [
-                {
-                  test: /\.js$/,
-                  exclude: /node_modules/,
-                  loader: 'babel-loader'
-                },
-                { test: /\.json$/, loader: 'json-loader' }
-              ]
-            },
-            plugins: [
-              // Ignore native modules (ed25519)
-              new webpack.IgnorePlugin(/ed25519/)
+    return gulp
+      .src('src/browser.js')
+      .pipe(plumber())
+      .pipe(
+        plugins.webpack({
+          output: { library: 'StellarSdk' },
+          module: {
+            loaders: [
+              {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                loader: 'babel-loader'
+              },
+              { test: /\.json$/, loader: 'json-loader' }
             ]
-          })
-        )
-        // Add EventSource polyfill for IE11
-        .pipe(
-          plugins.insert.prepend(
-            fs.readFileSync(
-              './node_modules/event-source-polyfill/src/eventsource.min.js'
-            )
-          )
-        )
```

Comment:
Diff is tricky to read, it looks like this is the only change? 👍 on removing polyfill